lib/oban.ex

```diff
@@ -647,6 +647,43 @@ defmodule Oban do
     |> Query.retry_job(job_id)
   end
 
+  @doc """
+  Retries all discarded and retryable jobs. For more complex mass retries, see `retry_all_jobs/2`.
+
+  ## Example
+
+      Oban.retry_all_jobs()
+      :ok
+  """
+  @doc since: "2.8.0"
+  @spec retry_all_jobs :: :ok
+  def retry_all_jobs() do
+    __MODULE__
+    |> config()
+    |> Query.retry_all_jobs()
+  end
+
+  @doc """
+  Retries all jobs that match on the given queryable.
+
+  ## Example
+
+      # Retries all jobs with priority 0
+      Oban.Job |> Ecto.Query.where(priority: 0) |> Oban.retry_all_jobs()
+      :ok
+  """
+  @doc since: "2.8.0"
+  @spec retry_all_jobs(name :: atom(), queryable :: Ecto.Queryable.t()) :: :ok
+  def retry_all_jobs(name \\ __MODULE__, queryable) do
```

Comment:
Since the `queryable` needs to have a state condition applied, this can default to `Oban.Job` (aliased as `Job`) instead:

```suggestion
  def retry_all_jobs(name \\ __MODULE__, queryable \\ Job) do
```